spec/amp-html-format.md

```diff
@@ -256,6 +256,8 @@ The following @-rules are allowed in stylesheets:
 
 Authors may add custom styles to a document using a single `<style amp-custom>` tag in the head of the document.
 
+In addition, a single `<style amp-keyframes>` tag can be used for `@keyframes` declarations in the end of the document. See [Keyframes stylesheet](#keyframes-stylesheet) section for more details.
```

Comment:
As we currently allow `@keyframes` CSS declarations in the regular `<style amp-custom>` tag, why do we need this new style tag?  It'd be worthwhile to put that info in here.

If we're recommending to use keyframes in this new style tag, consider rewriting this area as follows: 

For `@keyframes` CSS declarations, add those rules to a `<style amp-keyframes>` tag, which must be located at the end of the AMP document. For details, see the [Keyframes stylesheet](#keyframes-stylesheet) section of this document.

Reply:
Rewrote according to your advise, but with some clarifications. Please PTAL again.